sqrl-tools/sqrl-cli/Dockerfile

```diff
@@ -1,15 +1,25 @@
 FROM eclipse-temurin:11
```

Comment:
Let's fix this docker image to a more specific version so the postgres dependency remains consistent over time

Reply:
okay. you mean to fix the version of eclipse-temurin base image ? which specific version do you suggests ? thanks

Reply:
I think it's [11.0.22_7-jdk](https://hub.docker.com/layers/library/eclipse-temurin/11.0.22_7-jdk/images/sha256-8aa05793d70544e950e50f35f3c97c8a7ba369bf132fb28364bb9d34e1129478?context=explore)

Reply:
done